htmd/parameterization/esp.py

```diff
@@ -201,24 +210,27 @@ def __init__(self):
 
         self._reciprocal_distances = None
 
+        self.equivalent_atom_groups = None
```

Comment:
Call equivalent atom detector internally, so we don't need this ugly attributes

Reply:
See other comment on class redesign. I'll leave it for the moment like this. We can discuss it tomorrow